src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -33,12 +31,6 @@
     @FXML
     private Label id;
     @FXML
-    private Label phone;
-    @FXML
-    private Label address;
-    @FXML
-    private Label email;
-    @FXML
     private FlowPane tags;
```

Comment:
Perhaps we can remove this variable since tags are not required to be shown for students for v1.2?

Reply:
thanks!